### PR TITLE
Fix exceptions when not auto importing

### PR DESCRIPTION
**Pull request by scohen**
I added additional unit tests, but they won't work with travis due to this being ahead of master and it requiring the as-yet-unannounced thrift 0.10.

@jparise

### CODE REVIEW

File: lib/riffed/server.ex [hunk 1]
```diff
@@ -206,13 +218,14 @@ defmodule Riffed.Server do
...
+    are_structs_auto_imported = Module.get_attribute(env.module, :auto_import_structs) or false
     function_handlers = Enum.map(functions,
       fn({fn_name, delegate}) ->
-        build_handler(thrift_meta, struct_module, fn_name, delegate, overrides.functions) end)
+        build_handler(thrift_meta, struct_module, fn_name, delegate, overrides.functions, are_structs_auto_imported) end)
 
     structs_keyword = ThriftMeta.Meta.structs_to_keyword(thrift_meta)
 
-    struct_module = if Module.get_attribute(env.module, :auto_import_structs)  do
+    struct_module = if are_structs_auto_imported do
```
Comment (jparise) [inline lib/riffed/server.ex]: Would read better if this was named `struct_are_auto_imported` or `auto_imported_structs`.

File: lib/riffed/struct.ex [hunk 2]
```diff
@@ -353,6 +353,19 @@ defmodule Riffed.Struct do
     |> Enum.partition(&MapSet.member?(struct_names, &1))
   end
 
+  def build_struct_finders(this_module, module_mapping) do
+    module_mapping
+    |> Enum.map(fn {source_name, dest_name} ->
+
```
Comment (jparise) [inline lib/riffed/struct.ex]: Drop this empty line?

File: lib/riffed/struct.ex [hunk 3]
```diff
@@ -353,6 +353,19 @@ defmodule Riffed.Struct do
...
+      dest = Module.concat(this_module, dest_name)
+      quote do
+        def destination_module(unquote(source_name)) do
```
Comment (jparise) [inline lib/riffed/struct.ex]: Will `defp` work here?

Comment (scohen) [inline lib/riffed/struct.ex]: No, I'm calling this outside of the module.

**Pull request merged**